pygeos/tests/test_io.py

```diff
@@ -375,17 +375,24 @@ def test_to_wkb_srid():
 
 
 @pytest.mark.skipif(
-    pygeos.geos_version >= (3, 8, 0), reason="Pre GEOS 3.8.0 has 3D empty points"
+    pygeos.geos_version < (3, 9, 0),
+    reason="Pre GEOS 3.9 empty point serialization is buggy",
```

Comment:
Just to ensure I am correctly reading this: you removed testing for this for GEOS <= 3.8? (which I am fine with, given the version dependent behaviour, as long as we can guarantee correct and stable behaviour starting with GEOS 3.9)

Reply:
(although there is now a whole block of code in geos.c between `#if !GEOS_SINCE_3_9_0` / `#endif` that isn't really tested at all? Or is it still tested through other tests?)

Reply:
You’re right, there should be *some* testing in GEOS<3.9, now the code isn’t touched anymore in the unittests.

Not sure what exactly to test. I don’t like “confirming faulty behaviour” which the previous tests did. Maybe just run the function and make sure it doesn’t segfault. I’ll remove the skip to see what exactly we have to do on earlier versions.